internal/core/application/operator_service.go

```diff
@@ -161,8 +161,8 @@ func (o *operatorService) DepositMarket(
 		numOfAddresses = 1
 	}
 
-	addresses := make([]string, 0, numOfAddresses)
-	blindingKeys := make([][]byte, 0, numOfAddresses)
+	list := make([]AddressAndBlindingKey, numOfAddresses, numOfAddresses)
```

Comment:
why three arguments?

Reply:
it's for setting the length of the array already... this way in the loop we don't use append(), but directly assign a value to list[i]